coldfront/core/project/forms.py

```diff
@@ -164,6 +164,7 @@ class ProjectReviewUserJoinForm(forms.Form):
     role = forms.CharField(max_length=30, disabled=True)
     auto_approval_time = forms.DateTimeField(disabled=True)
     selected = forms.BooleanField(initial=False, required=False)
+    reason = forms.CharField(max_length=100, required=False, disabled=True)
```

Comment:
When denying a join request, I saw the following error:

![error](https://user-images.githubusercontent.com/10734545/116896148-c91fd100-abe8-11eb-80bd-7f01ce24e65e.png)

The form for filling in the reason can be arbitrarily long, but the field in the review form is capped at 100. I think we can make the limit larger (1000), but also enforce it on the request side.

Reply:
alright, and what about the min length? currently the form doesn't submit for reason fields within 50 characters

Reply:
I think 20 is fine for a min length, which is what you defined in the models.